content/applications/sales/crm/optimize/gamification.rst

```diff
@@ -1,86 +1,159 @@
+  :guilabel:`Required Badges`. Use this drop-down to select the badges a user must have before they
+  can award this badge to others.
+- :guilabel:`No one, assigned through challenges`: this badge cannot be manually granted, it can
+  only be awarded through challenges.
+
+To limit the number of badges a user can send, tick the :guilabel:`Monthly Limited Spending`
+checkbox. This sets a limit on the number of times a user can grant this badge. In the
+:guilabel:`Limitation Number` field, enter the maximum number of times this badge can be sent per
+month, per person.
+
+.. image:: gamification/create-badge.png
+   :align: center
+   :alt: The details page for a new badge.
 
-| The users can be evaluated using goals and numerical objectives to reach.
-| **Goals** are assigned through *challenges* that evaluate (see image above) and compare members of
-  a team throughout time.
-| You can create a new goal on the fly from a *Challenge* by clicking on *Add new item*, under
-  *Goals*. Select the business object according to your company's needs. Examples: number of new
-  leads, time to qualify a lead, total amount invoiced in a specific week, month or any other time
-  frame based on your management preferences, etc.
+.. _crm/create-challenge:
 
-.. image:: gamification/goal_definitions.png
-   :align: center
-   :alt: View of the goal definition form and a goal definition being created for Odoo Sales
+Create a challenge
+==================
 
-.. note::
-   Goals may include your database setup as well (e.g. set your company data and a timezone,
-   create new users, etc.).
+To create a challenge, navigate to to :menuselection:`Settings --> Gamification Tools -->
+Challenges`. Click :guilabel:`New` in the top-left corner to open a blank challenge form.
 
-Set up rewards
---------------
+At the top of the form, enter a :guilabel:`Challenge Name`.
 
-For non-numerical achievements, **Badges** can be granted to users. From a simple thank you to an
-exceptional achievement, a badge is an easy way to express gratitude to a user for their good work.
-To create and grant badges to your employees based on their performance, go to
-:menuselection:`Settings --> Gamification Tools --> Badges`.
+Create assignement rules
+------------------------
 
-.. image:: gamification/badges.png
+To assign the challenge to specific users, one or more assignment rules must be utilized.
+
+Click into the first field under :guilabel:`Assign Challenge to`, and select a parameter from the
+drop-down list to define the rule. Then, click into the next field to define the rule's operator. If
+necessary, click into the third field to further define the parameter.
+
+.. tip::
+   To include all users with permissions in the *Sales* app, create a rule with the following
+   parameters:
+   - :guilabel:`Groups`
+   - :guilabel:`is in`
+   - `Sales/User: Own Documents Only`
+
+   .. image:: gamification/assignation-rule.png
```

Comment:
The way the condition is showing in the field is a display bug (it should be flush with the row), so let's retake and zoom out slightly or increase browser window size.